frontend/public/components/utils/webhooks.tsx

```diff
@@ -55,10 +55,51 @@ export const WebhookTriggers: React.FC<WebhookTriggersProps> = (props) => {
     namespace,
   });
   const tableColumnClasses = getTableColumnClasses(canGetSecret);
-  const webhooks = _.filter(triggers, (trigger) => webhookTriggers.has(trigger.type));
+  const [webhookSecrets, setWebhookSecrets] = React.useState<K8sResourceKind[]>([]);
+  const [secretErrors, setSecretErrors] = React.useState<string[]>([]);
+  const [isLoaded, setIsLoaded] = React.useState(false);
+
+  const webhooks: WebhookTrigger[] = _.filter(triggers, (trigger) =>
```

Comment:
nit: you can use es6 syntactic sugar

```suggestion
  const webhooks: WebhookTrigger[] = _.filter(triggers, ({type}) => 
    webhookTriggers.has(type),
  );
```